int8 meta3
